guide/ops/requirements.md

```diff
@@ -19,6 +19,43 @@ at least 8GB RAM and 100GB disk. The disk is just for logs, a small amount of pe
 any binaries for custom blueprints/integrations.
 
 
+### Disk Space
+
+There are three main consumers of disk space:
+
+* **Static files**: these are the Apache Brooklyn files themselves, plus
+  binaries for custom blueprints and integrations added to the `lib` directory.
+* **Persisted state**: when using [Persistence](persistence/index.html) -- which
+  is a prerequisite for [High Availability](high-availability.html) -- Brooklyn
+  will save data to a store location. Items in the persisted state include
+  metadata about the Brooklyn servers, catalog items, and metadata about all
+  running applications and entities.
+* **Log files**: Brooklyn writes detailed log files by default to its own
+  installation directory. This can be reconfigured to change the destination
```

Comment:
Agreed. Perhaps we don't need to say here where it's written to - you've already linked to the logging section. Instead just say:

     **Log files**: Brooklyn writes info and debug log files. By default, these are 
    written to the local filesystem. This can be reconfigured to set the destination
    and to increase or decrease the detail in the logs. See the
    [Logging](logging.html) section for more details.